### PR TITLE
{bio}[GCCcore/11.3.0] MotionCor2 v1.5.0

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/403e2b26bd793125e626128cbd0e8c68 for a full test report.

File: easybuild/easyconfigs/m/MotionCor2/MotionCor2-1.5.0-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Cite publication: Shawn Q. Zheng, Eugene Palovcak, Jean-Paul Armache,
+Yifan Cheng and David A. Agard (2016) Anisotropic Correction of
+Beam-induced Motion for Improved Single-particle Electron
+Cryo-microscopy, Nature Methods, submitted.
+BioArxiv: https://biorxiv.org/content/early/2016/07/04/061960
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+# No longer directly downloadable, available from https://msg.ucsf.edu/software
```
Comment (branfosj) [inline easybuild/easyconfigs/m/MotionCor2/MotionCor2-1.5.0-GCCcore-11.3.0.eb:24]:
```suggestion
```

File: easybuild/easyconfigs/m/MotionCor2/MotionCor2-1.5.0-GCCcore-11.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+sources = [
+    '%(name)s_%(version)s.zip',
+]
+checksums = ['cdb7b9aceb42d52e2a4c7a5e80270371376aea412d21fa1f1f28d4cc71c81834']
+
```
Comment (branfosj) [inline easybuild/easyconfigs/m/MotionCor2/MotionCor2-1.5.0-GCCcore-11.3.0.eb:29]:
```suggestion
download_instructions = """Visit https://msg.ucsf.edu/software and fill in the form to download %s""" % sources[0]

```

Adding this means we get the following in the output in the fetching files step if you do not have that file downloaded:
```console
$ eb MotionCor2-1.5.0-GCCcore-11.3.0.eb -r
== fetching files...

Download instructions:

Visit https://msg.ucsf.edu/software and fill in the form to download MotionCor2_1.5.0.zip


```

**Review by branfosj — CHANGES_REQUESTED** [review]
Add `download_instructions` to improve the experience if you do not already have the relevant file downloaded.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0103u11a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6330 CPU @ 2.00GHz (icelake), 1 x NVIDIA NVIDIA A100-PCIE-40GB, 470.57.02, Python 3.6.8
See https://gist.github.com/ac58cfdea13926738305d901a2dacda0 for a full test report.

**Review by branfosj — APPROVED** [review]

**Pull request merged**